src/content/docs/journey-large-logs/organize.mdx

```diff
@@ -0,0 +1,196 @@
+*   Create data partitions that align with concepts in your environment or organization that are static or change infrequently (for example, by business unit, team, environment, service, etc.).
+*  Ensure each partition remains below 1 TB of daily ingest for optimal performance.
+
+There are two types of retention for partitions. Choose **Standard** for maximimum retention; use this for valuable data you may want to query far in the future. Choose **Secondary** for 30-day retention; use this for less valuable data or for data you will only need in the near future. 
+
+## Why parse logs
+
+Parsing your logs at ingest is the best way to make your log data more usable by you and other users in your organization. For example, compare these two logs pre-parsing and post-parsing using a Grok parsing rule:
+
+<SideBySide>
+    <Side>
+Pre-parsing:
+
+```
+{
+    "message": "32 4329 store_Portland"
+}
+
+
+```
+
+    </Side>
+    <Side>
+
+Post-parsing:
+
+```
+{
+    "transaction_total": "32",
+    "purchase_number": "4329",
+    "store_location": "store_Portland",
+}
+```
+
+    </Side>
+</SideBySide>
+
+
+
+This allows you to easily query for newly defined attributes such as `transaction_total` in dashboards and alerts.
+
+
+
+## Organize your logs
+
+The following steps will guide you through how to organize your logs in the New Relic UI.
+
+<Steps>
+    <Step>
+    ### Navigate to the UI
+
+    Go to **[one.newrelic.com > Logs](https://one.newrelic.com/launcher/logger.log-launcher)**    
+    </Step>
+    <Step>
+
+### Partition your logs
+
+Let's say ACME Corp knows they will ingest about 2 TBs of logs in the coming months. They want to create partitions for logs coming from their Java app and from their infrastructure agent. They think they may need to query their Java logs far into the future so they decide to use a standard retention. Meanwhile, they only need recent infrastrucutre logs so they will use secondary retention for those.
+
+To do so:
+
+<SideBySide>
+    <Side>
+
+1. From **Manage data** on the left nav, click **Data partitions**, then click **Create partition rule**.
+2. Define a Partition name as an alphanumeric string that begins with Log_. In this case `Log_java`. 
+3. Add an optional description.
+4. Select the standard namespace retention for the partition.
+5. Set your rule's Matching criteria: Enter a valid NRQL WHERE clause to match the logs to store in this partition. In this case `logtype=java`.
+6. Click Create to save your new partition.
+
+    </Side>
+    <Side>
+
+<img
+    title="log-partition"
+    alt="An image displaying New Relic's log partion UI"
+    src={logsPartition}
+/>
+
+    </Side>
+</SideBySide>
+
+
+
+
+This creates a data partition with standard data retention for all Java logs. They would follow the same steps for their infrastrucutre logs, only changing the retention to secondary and query to `logtype=infrastructure`.
+
+    </Step>
+    <Step>
+
+### Parse your logs
+
+To parse your logs:
```

Comment:
the previous step has an in-depth intro ¶, suggest something similar is needed here.